Libplanet.Tests/TestUtils.cs

```diff
@@ -161,9 +161,11 @@ HashDigest<T> actual
 
         public static byte[] GetRandomBytes(int size)
         {
-            var random = new System.Random();
             var bytes = new byte[size];
-            random.NextBytes(bytes);
+            using (RandomNumberGenerator rng = RandomNumberGenerator.Create())
+            {
+                rng.GetBytes(bytes);
+            }
```

Comment:
We could use [C# 8's `using` declaration][1] here.

```suggestion
            using RandomNumberGenerator rng = RandomNumberGenerator.Create();
            rng.GetBytes(bytes);
```

[1]: https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-8#using-declarations